sotodlib/flags.py

```diff
@@ -83,3 +83,46 @@ def get_glitch_flags(tod, params={}, signal='signal', merge=True,
             tod.flags.wrap(name, flag)
         
     return flag
+
+def get_trending_flags(aman, max_trend=np.pi, n_pieces=1, signal_name='signal',
+                 flag_name='trends', overwrite=True):
```

Comment:
This further entrenches the old aman API conventions (i.e. signal_name='signal' instead of signal=None).  I know you plan to update everything ... someday ... but consider just starting to develop with the new API.